All Core Devs Meetings/Meeting 82.md

```diff
@@ -0,0 +1,735 @@
+
+Ameen Soleimani asked about what is required to become a core dev.  Hudson indicated we can talk about this off line.
+
+Tim Beiko and others again asked for clarification of the current state of the EIP.
+
+Hudson Jameson replied: “It is currently in accepted state and nothing has been changed on that.  There has been a suggestion to make this what is known in the US as a “pocket veto” and then to supersede it with another EIP, possibly called Ethash2, that are improved, memory hard, etc.  These are only suggestions, and there is no further movement on the EIP status, today.
+
+Martin Holst Swende commented that there were “core dev code of conduct” violations, and that this is not acceptable.  Hudson Jameson agreed.  He will be talking to Ameen and putting in more things to prevent this in the future.  Armeen commented that he wanted to better understand, as he thought he was showing restraint.
+
+Matt Lounge thanked everyone for having this discussion in this call.
+
+**Kristy-Leigh Minehan**
+I just have a few questions to ask, anyone can answer via twitter or eth-magicians.  I want to know what the pro ProgPow community needs to do so we can be more helpful.  Guidelines on what education would be helpful to the broader community as a whole would help.  The whole point of ProgPOW is ASIC resistance.  I would like to have more of a discussion on whether we want ASIC resistance or not, which stems back to the principle in the original yellow paper.  This needs to be answered by the community before more investment.
+
+**Martin Koppelmann**
+I would like to flip this question and ask what the community needs to do to get the ProgPOW people to no longer push it.
+
+**Stefan George**
+I would like to add to this, that you may be underestimating the community a bit.  There has already been a lot of education in the last weeks.  People signing the petition are well aware of all arguments that have been made.  It really comes down to the question of whether we want to be ASIC resistant or not.  The people signing the petition are willing to take the risk.  I am worried that this could go on for a very long time and be a waste of time.
+
+**Kristy-Leigh Minehan**
+The [“EIP 2538” letter](https://github.com/MidnightOnMars/EIPs/blob/master/EIPS/eip-2538.md) (not a real EIP) missed a lot of the arguments.  It’s not about favoring anyone, it’s about leveling.  There hasn’t been enough education...
+
+**James Hancock**
+I clarified that part Kristy.  Trent was saying there hasn’t been education on the vulnerability of Ethash.
+
+**Kristy-Leigh Minehan**
+Got it.
+
+**Hudson Jameson**
+Thanks everyone for the stuff that has been said, I think there needs to be more discussion.  We’ve made progress since the beginning of the meeting.  For everyone that has just joined, there is agreement that ProgPow Isn’t going in any time so
```

Comment:
This is Vitalik Buterin

Reply:
That section you highlighted should be Hudson. https://youtu.be/kham8c0qhmw?t=9490

But, all cases of the person Vub should be Vitalik.